spec/core/genesis.md

```diff
@@ -0,0 +1,32 @@
+# Genesis
+
+The genesis file is meant to be the starting point of chain. Tendermint's genesis is minimal because it is not aware of application state. An application will populate the genesis file with their own required fields.
+
+## Genesis Fields
+
+- `genesis_time`: The genesis time is the time the blockchain is meant to start. If a nodes are started before this time they will sit idle until time specified.
+- `chain_id`: The chainid is the chain identifier. Every chain should have a unique identifier. When conducting a fork based upgrade, we recommend changing the chainid to avoid network or consensus errors.
+- `initial_height`: This field is the starting height of the blockchain. When conducting a chain restart to avoid restarting at height 1, the network is able to start at a specified height.
+- `consensus_params`
+    - `block`
+        - `max_bytes`: The max amount of bytes a block can be.
+        - `max_gas`: The maximum amount of gas that a block can have.
+        - `time_iota_ms`: This parameter has no value anymore in Tendermint-core.
+
+- `evidence`
+      - `max_age_num_blocks`: After this preset amount of blocks has passed a single piece of evidence is considered invalid
+      - `max_age_duration`: After this preset amount of time has passed a single piece of evidence is considered invalid.
+      - `max_bytes`: The max size a single piece of evidence can be.
```

Comment:
This is the max possible size of all evidence in a block

Reply:
oh, whoops thanks.